src/config/graphical-session/graphics-drivers/nvidia.md

```diff
@@ -0,0 +1,64 @@
+# Nvidia GPU
+
+## nouveau (Open Source drivers)
+
+Install the Mesa DRI drivers and Xorg Drivers:
+
+```
+# xbps-install -S mesa-nouveau-dri xf86-video-nouveau
+```
+
+> Note: This is already included in the `xorg` metapackage, but it is needed
+> when installing xorg via `xorg-minimal` or for running a Wayland compositor.
```

Comment:
You shouldn't need xf86 drivers for Wayland afaik. Nouveau crashes on me when trying to use Wayland, but I will try to give it a go.

Reply:
Should I create a separate section for Wayland?

Reply:
I don't think we need that, the Wayland page already explains it pretty well.